apps/frontend/src/libs/enums/notification-message.enum.ts

```diff
@@ -1,5 +1,6 @@
 const NotificationMessage = {
-	CONTRIBUTOR_MERGE_SUCCESS: "Contributor was successfully merged.",
+	CONTRIBUTOR_MERGE_SUCCESS: "Contributors were successfully merged.",
+	CONTRIBUTOR_SPLIT_SUCCESS: "Contributors were successfully split.",
```

Comment:
In case you choose plural here, then change `CONTRIBUTOR_SPLIT_FAILED` to plural as well:
"Failed to split contributors." instead of "Failed to split contributor." 

or leave `CONTRIBUTOR_SPLIT_FAILED` without any change, and change this message to "Contributor wes successfully split."